code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -94,6 +94,14 @@ func GetEndpoints(rh rest.RestHandlerI) []rest.Endpoint {
 		rest.MakeEndpoint(storage+"/replicate-authorizer-aggregates", srh.replicateAuthorizerAggregates),
 		rest.MakeEndpoint(storage+"/replicate-validator-aggregates", srh.replicateValidatorAggregates),
 		rest.MakeEndpoint(storage+"/replicate-user-aggregates", srh.replicateUserAggregates),
+		rest.MakeEndpoint(storage+"/reward-providers", srh.rewardProviders),
```

Comment:
do we need all this handlers public? let's hide them under development/test urls